scripts/circle-ci/install-docker.sh

```diff
@@ -2,9 +2,20 @@
 
 set -x
 
-MIN_VERSION_WANTED=1.10.0
 CURRENT_VERSION=$(docker version --format '{{.Client.Version}}')
 
+if [[ $INTERNAL_BUILD == true ]]; then
+    MIN_VERSION_WANTED=1.13.1
```

Comment:
Discussed offline: Internal CircleCI seems to support this, whilst external can't seem to go beyond 1.10.0. We need 1.12 for native healthchecks. 

I agree that this separation isn't great, but holding back features because of external Circle being old also doesn't seem right to me - so opted to go via this route. We can revisit this in the future if necessary.